### ISSUE TITLE
Search results always empty

### ISSUE THREAD

**Issue by ceesvanegmond** [whatwedo/SearchBundle#16]
Hi!,

I've installed this bundle; no problem what so ever. Also indexing is good, migrated properly.
But when i try to search, the results are always empty. I've debugged the code, and it seems that `:minscore` in the QueryBuilder is not working properly. For example,  in the `whatwedo_search_index` table there is a row with `content` 'home'.

When searching on 'home' using the IndexRepository, zero results.

```php

 public function indexAction()
    {
        $results = $this->searchRepository->search('home', Page::class);

        dd($results);
     }
```

When commenting the minscore in the query builder, I do get the result. The `minScore` for home is 6 (strlen('home') * 1.5). But the `minscore` of the actual result is 5.5. 

Any idea? Using MySQL 5.7 here.

**Comment by lukasluecke** [comment]
Can you try reverting the bundle to `v0.1.12` and testing if it works there? If so, these issues in `v0.2.0` should be already fixed for the next release, which will happen sometime next week.

**Comment by ceesvanegmond** [comment]
@lukasluecke Yes that works! What seems to be the problem?

**Comment by lukasluecke** [comment]
The 0.2 release was pushed prematurely, and the fixes weren‘t released since then - sorry for that! I will ping you again once a working release is available again, until then the 0.1 release should work just fine :+1:

**Comment by lukasluecke** [comment]
@ceesvanegmond We just released v2.0.1 which should fix the issues, please let us know if there's anything else!

**Comment by xarem** [comment]
> @lukasluecke Almost! See #17

We just released v2.0.2 - thanks.
